Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandCharacterEquipSet.java

```diff
@@ -98,7 +98,7 @@ public int compare(CachedEntity e1, CachedEntity e2) {
 
 		for (CachedEntity equipment : setEquip) {
 
-			if (slotList.size() == 0) {
+			if (slotList.size() == 0 || slotList==null) {
```

Comment:
Actually, it never is going to be null at that point, since I initialized it before. That is a leftover of a thinking process, sorry. Will remove.